src/snapshot.ts

```diff
@@ -207,8 +144,40 @@ export class Snapshot {
   static formatName_(projectId: string, name: string) {
     return 'projects/' + projectId + '/snapshots/' + name.split('/').pop();
   }
+
+
+  create(gaxOpts?: CallOptions): Promise<CreateSnapshotResponse>;
+  create(callback: CreateSnapshotCallback): void;
+  create(gaxOpts: CallOptions, callback: CreateSnapshotCallback): void;
+  create(
+      gaxOpts?: CallOptions|CreateSnapshotCallback,
+      callback?: CreateSnapshotCallback): void|Promise<CreateSnapshotResponse> {
+    if (!(this.parent instanceof Subscription)) {
+      throw new Error(`Subscription#snapshot`);
+    }
+    return (this.parent as Subscription)
+        .createSnapshot(this.name, gaxOpts! as CallOptions, callback!);
+  }
+
+  seek(gaxOpts?: CallOptions): Promise<google.pubsub.v1.SeekResponse>;
+  seek(callback: google.pubsub.v1.Subscriber.SeekCallback): void;
```

Comment:
I'm not sure if we should be using the generated callbacks (anywhere ever). They all look to return the proto class version of the response which we don't use. I think instead we'll want something like

```ts
type SeekCallback = RequestCallback<google.pubsub.v1.Subscriber.ISeekResponse>;
```

Reply:
Done